pep-0636.rst

```diff
@@ -0,0 +1,44 @@
+PEP: 636
+Title: Structural Pattern Matching: Tutorial
+Version: $Revision$
+Last-Modified: $Date$
+Author: Daniel F Moisset <dfmoisset@gmail.com>,
+        Tobias Kohn <kohnt@tobiaskohn.ch>
```

Comment:
You should probably list yourself as PEP 636's sponsor.

Reply:
Do you have time to make that change while I do other stuff?

Reply:
Yep.